components/Navbar/NavbarMobile/HamburgerMenu.tsx

```diff
@@ -0,0 +1,41 @@
+import clsx from 'clsx'
+import React, { useState } from 'react'
+
+export default function HamburgerMenu(props) {
+  const aria_label = props.isOpen ? 'Close Menu' : 'Open Menu'
```

Comment:
Please use camelCase for variable names

Reply:
Renamed it in 9b4c5743cb1375d2eae011ab956848b2f92682c6